daweb/js1/cykly/cvlekce/pole-v-divadle.md

```diff
@@ -1,11 +1,41 @@
 ---
 title: Pole v divadle
 demand: 1
+solutionAccess: lock
 ---
 
-Celé toto cvičení prováďejte v JavaScript konzoli.
+Celé toto cvičení provádějte v JavaScript konzoli.
```

Comment:
Opravdu tady chceme tohle nechat? Myslel jsem, že chceme „cvičení v konzoli“ eliminovat úplně, i když už účastnice konzoli znají.
Podle mne jsou cvičení v konzoli matoucí, zejména když tam napíšou něco špatně. Chrome už umožňuje v konzoli opakovaně vytvářet přes `const`/`let` stejnou proměnnou, ale jiné prohlížeče to asi neumí. Nebo dají refresh stránky a o všem přijdou. Stejně si to účastnice chtějí uložit do nějakého souboru.
Za sebe bych konzoli používal opravdu jenom pro ladění programu, žádné programování v konzoli.

Reply:
diky upravil jsem instrukce na provadeni v JS souboru misto konzole :)